src/index.tsx

```diff
@@ -257,11 +256,9 @@ render(
         <DndProvider backend={HTML5Backend}>
           <Provider store={store}>
             <ReactReduxFirebaseProvider {...rrfProps}>
-              <CustomSoundsProvider>
-                <AuthIsLoaded>
-                  <AppRouter />
-                </AuthIsLoaded>
-              </CustomSoundsProvider>
+              <AuthIsLoaded>
+                <AppRouter />
+              </AuthIsLoaded>
```

Comment:
> I assume the whole app should not be wrapped by the secure provider `<CustomSoundsProvider/>` so far as we watch a lot of errors in Bugsnap for not logged users

From https://github.com/sparkletown/internal-sparkle-issues/issues/591:

> A small fix was made in sparkletown/sparkle#1279 to ensure that we don't try and fetch the sounds till after AuthIsLoaded has finished running, as the security rules we have configured require a user to be authenticated before the can fetch the sounds data.

@yurii-lubynets Curious why you chose to move `CustomSoundsProvider` into `src/pages/VenuePage/VenuePage.tsx` in this PR, rather than the method used in #1279?

Reply:
Yeah, I strongly agree with it. Like it's better to keep just inside `<AuthIsLoaded/>`